GPU/GPUTracking/Base/GPURawData.h

```diff
@@ -56,16 +58,16 @@ GPUdi() unsigned int GPURawDataUtils::getOrbit(const o2::header::RAWDataHeader*
 #endif
 }
 
-GPUdi() unsigned int GPURawDataUtils::getBC(const o2::header::RAWDataHeader* rdh)
+GPUdi() unsigned int GPURawDataUtils::getBC(const RAWDataHeaderGPU* rdh)
 {
 #ifndef __OPENCL__
   return o2::raw::RDHUtils::getHeartBeatBC(*rdh);
 #else
-  return ((rdh->words[4] >> 48) & 0xFFF); // TODO: Ad-hoc implementation for OpenCL, RDHV4, to be moved to RDHUtils
+  return (rdh->words[2] & 0xFFF);         // TODO: Ad-hoc implementation for OpenCL, RDHV4, to be moved to RDHUtils
```

Comment:
the comments still refer to RDHV4, perhaps should be removed completely

Reply:
No, it's unfortunately not only the bitsets. In order to automatically use the correct implementation, you have implemented this RDHAny class, and there are void* versions of the getters taking an arbitrary RDH. Unfortunately, RDHAny involves std:: stuff, which doesn't work either.

Reply:
RDHAny is needed only for operations on arbitrary RDH references (also pointers). In the methods with ``...(void* rdhP)`` it is not involved. One can feed to them any pointer (void or RAWDataHeaderVX* cast with whathever X) and it should work, at least this was the idea.

Reply:
OK, then I don't fully remember. But there was a problem. Perhaps it was with the std::enable_if in RDHUtils.